feat: add examples implementation and documentation (#3)
terraform-mongodbatlas-modules/terraform-mongodbatlas-push-based-log-export: examples/s3bucket-create_bucket_generate_name/README.md

> # push-based-log-export-s3_bucket - create a new bucket with a default name

Reviewer: Both for the title and name of directories in examples would be consistent with using snake_case or dash-case. Personally like more dash case and see it used in [aws module](https://github.com/terraform-aws-modules/terraform-aws-s3-bucket/tree/master/examples) as well.

> This example shows how you can use the push-based-log-export-s3_bucket submodule to enable push-based logging in an Atlas project using a new s3 bucket that has a default name.

Reviewer: ```suggestion
This example shows how you can use the s3 bucket submodule to enable push-based logging in an Atlas project. The module will create a new s3 bucket with a default name.
```
Just making more explicit the fact that the s3 bucket is being created as part of the module. Suggestion also applies to second example.
